xml/docu_styleguide.structure.xml

```diff
@@ -1420,10 +1420,19 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
   <para>To support automatic alphabetical ordering in localized versions, use
     <tag>glosslist</tag> so that glossary items are sorted in alphabetical order by default.</para>
   <para>
-   Use cross-references to link acronyms with their written out forms.
-   Define the written out form. Use a <emphasis>See also</emphasis> reference for
-   the acronym form to link it to the defined written out form.
-  </para>
+   Use cross-references in the following cases:
+   </para>
+  <itemizedlist>
+    <listitem>
+      <para>To <emphasize>replace</emphasize> the existing glossary to make the reader aware of reading another glossary entry, use <tag>glosssee</tag>.
```

Comment:
```suggestion
      <para>To direct the reader to another glossary entry, use the <tag>glosssee</tag> tag. This can be used, for example, to link acronyms with their written out forms.
```